maven-core/src/test/java/org/apache/maven/project/ProjectBuilderTest.java

```diff
@@ -84,4 +88,36 @@ public void testVersionlessManagedDependency()
             // this is expected
         }
     }
+
+    public void testReadModifiedPoms() throws Exception {
```

Comment:
This entire method does not adhere to our formatting rules.

Reply:
I'll fix it. But I'd like to use it as an opportunity to mention that formatting rules (particularly these ones) are not really making the code welcoming.

Reply:
I do agree, but we won't able to change them for now. Even if it is a bad format, it should be consistently bad.

Reply:
The need of consistency is also arguable IMHO. The real need is readability and consistency doesn't necessarily improve that.